### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by CapsAdmin**

### CODE REVIEW

**Comment by Tenrys** [comment]
Why is this needed??

**Comment by PotcFdk** [comment]
In my eyes, it is not.

**Comment by Etothepowerof26** [comment]
????

**Comment by GitPaulo** [comment]
Seems pointless - but there is no reason to not have it? *shrug*

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ * Posting or threatening to post other people's personally identifying information ("doxing").
+ * Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+ * Inappropriate photography or recording.
+ * Inappropriate physical contact. You should have someone's consent before touching them.
+ * Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances.
+ * Deliberate intimidation, stalking or following (online or in person).
+ * Advocating for, or encouraging, any of the above behavior.
+ * Sustained disruption of community events, including talks and presentations.
+
+## 5. Weapons Policy
```
Comment (danielga) [inline CODE_OF_CONDUCT.md]: Weapons policy???

**Comment by Zeni44** [comment]
stop using a template CoC and doing this meme for something completely irrelevant

**Comment by Earu** [comment]
Well this meme is boring already

**Pull request closed without merging**